app/src/main/java/org/p2p/wallet/App.kt

```diff
@@ -84,4 +87,12 @@ class App : Application() {
             setCustomKey("task_number", BuildConfig.TASK_NUMBER)
         }
     }
+
+    private fun setupAppsFlyer() {
+        val appsFlyer = AppsFlyerLib.getInstance().apply {
+            setDebugLog(BuildConfig.DEBUG)
+        }
+        appsFlyer.init(BuildConfig.appsFlyerKey, null, this@App)
```

Comment:
please try to put appsflyer into a Service object like Intercom, we need abstraction here

Reply:
also we can try and put the init block into a try catch - to remove the possibility of crashes if Appsflyer goes wrong

Reply:
Got you